src/components/Calculator.js

```diff
@@ -0,0 +1,59 @@
+/* eslint linebreak-style: 0 */
+import React, { Component } from 'react';
+import './Calculator.css';
+import { Button } from './Button';
+import { Input } from './Input';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      input: '',
+    };
+  }
+
+  render() {
+    return (
+      <div className="app">
+        <div className="calc-wrapper">
+          {/* eslint-disable-next-line react/destructuring-assignment */}
```

Comment:
[Optional] As much as possible, kindly avoid disabling linter checks in line. They represent coding standards that you share with your team and most of the time enforce adherence to best practices.

Reply:
@emmiiorji Yes, I understand. I looked all over to try to find a solution to fix this, but I could not find the right syntax for this one. If you have any suggestions, please let me know so I can improve my code.

Thank you!

Reply:
Hi @Boku52500

Thanks for reaching out. You actually have several instances of disabling in line linter checks. However, in this case, I think that destructing the input will fix the error. E.g const { input } = this.state